compiler_opt/es/blackbox_optimizers.py

```diff
@@ -35,18 +32,16 @@
 import math
 import numpy as np
 from sklearn import linear_model
+from absl import flags
+import scipy.optimize as sp_opt
 
-import gradient_ascent_optimization_algorithms
+from compiler_opt.es import gradient_ascent_optimization_algorithms
 
 
 def filter_top_directions(perturbations, function_values, est_type,
```

Comment:
consider adding pytype information here, especially est_type = Union[Literal['forward_fd'], Literal['antithetic'], ...]

Reply:
I will add this via the type annotations pr